packages/api/controllers/nomination.js

```diff
@@ -14,10 +15,30 @@ const getNominationById = async (req, res) => {
     if (nomination) {
       return res.status(200).json({ nomination });
     }
-    return res.status(404).send('Nomination with the specified ID does not exist!');
+    return res
+      .status(404)
+      .send('Nomination with the specified ID does not exist!');
   } catch (error) {
     return res.status(500).send(error.message);
   }
 };
 
-module.exports = { getNominationById };
+const createNomination = async (req, res) => {
+  if (req.body === null) {
+    return res.status(404).send('Required fields are not completed');
```

Comment:
I think we'd want 400 here, right? Do we need special handling here, or would this be handled by the validation step on create?

Reply:
gotcha